woocommerce/woocommerce
# Fix obw free extension rules for the marketing task with php 8

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33216.

The Marketing task is showing a blank screen for a store that skips OBW with PHP 8.

In PHP 7, when a string is passed to array_column and count, it only shows a warning notice but PHP 8 throws a fatal error.

When a user skips OBW, the default value will be used.

### How to test the changes in this Pull Request:

1. Create a fresh site.
2. Make sure `woocommerce_show_marketplace_suggestions` is set to `no` in `wp_options` table.
3. Navigate to HTTP://your-site/wp-admin/admin.php?page=wc-admin&task=marketing 
4. Confirm the Marketing task is shown without API errors.


### Other information:




### FOR PR REVIEWER ONLY:
